src/styles/_headings.scss

```diff
@@ -0,0 +1,62 @@
+@use 'variables/ktzh-design-variables' as ktzh-variables;
+
+// This file is a collection of heading styles that are used across the application.
+// The names of the classes describe which heading each class can be used for.
+// The styles are based on the design system of the Kanton of Zürich:
+// (https://www.figma.com/design/MsFtPBJrdzPPevIAZwtxDn/GB3-mit-KTZH-Design-System?node-id=10-5055&node-type=canvas&t=BCmKrCIFSSdxjNgz-0)
+
+h4.ktzh-h4-bigger-regular {
+  margin: 0;
+
+  font-size: 18px;
+  font-family: ktzh-variables.$zh-font-family-bold;
+
+  letter-spacing: 0.25px;
+}
+
+h3.ktzh-h3-medium--bold {
+  font-size: 20px;
+  font-family: ktzh-variables.$zh-font-family-bold;
+}
+
+h3.ktzh-h3-medium--thin {
+  font-size: 20px;
+  font-family: ktzh-variables.$zh-font-family;
+}
```

Comment:
I had it like that because the "--thin" variant was only used once in the entire application, so I would only have to apply one classname to the once that are bold. But you are right, this is the cleaner way.